app/frontend/test/unit/specs/search/SearchTable.spec.js

```diff
@@ -80,30 +80,90 @@ describe('SearchTable.vue', () => {
     })
     expect(wrapper.find('#table-pagination-prev').text()).toEqual('Previous')
   })
-  it('dispatches fetch driller list with correct querystring when pagination next clicked', () => {
+  it('shows the paganation button disabled for previous page when no previous link is present in payload', () => {
+    let fakePersonListCopy = Object.assign({}, fakePersonList)
+    fakePersonListCopy.previous = null
+    getters = {
+      loading: () => false,
+      listError: () => null,
+      drillers: jest.fn().mockReturnValue(fakePersonListCopy),
+      userRoles: () => ({ registry: { edit: false, view: false, approve: false } }),
+      activity: () => 'DRILL'
+    }
+    store = new Vuex.Store({
+      getters,
+      actions
+    })
     const wrapper = shallowMount(SearchTable, {
       store,
       localVue,
       stubs: ['router-link', 'router-view']
     })
-    wrapper.find('#table-pagination-next').trigger('click')
-    expect(actions.FETCH_DRILLER_LIST.mock.calls[0][1]).toEqual({
-      limit: '30',
-      offset: '60'
+    expect(wrapper.find('#table-pagination-prev').attributes().disabled).toBe('disabled')
+  })
+  it('shows the paganation button disabled for next page when no next link is present in payload', () => {
```

Comment:
minor nitpick: pagination is spelled wrong a couple times (search all for 'paganation')